### PR TITLE
Fix/41 repeating values

### PR DESCRIPTION
**Pull request by jnizet**
This allows repeating the same row several times. It's useful to populate the database with a large amount of values, using value generators. We could already do that with a simple for loop, but the syntax is prettier:

```
Insert.into("tag")
      .withGeneratedValue("id", ValueGenerators.sequence().startingAt(1L))
      .withGeneratedValue("name", ValueGenerators.stringSequence("tag-").startingAt(1L))
      .columns("description")
      .repeatingValues("fake description").times(1000)
      .build();
```

Note that we can also pass a Map instead of a vararg for the values. 
Also note that name/value rows built with a row builder can also be repeated:

```
Insert.into("tag")
      .withGeneratedValue("id", ValueGenerators.sequence().startingAt(1L))
      .withGeneratedValue("name", ValueGenerators.stringSequence("tag-").startingAt(1L))
      .row()
          .column("description", "fake description")
          .times(1000)
      .build();
```

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1 +1 @@
-version = 1.5.0
+version = 1.6.0-SNAPSHOT
```
Comment (clacote) [inline gradle.properties]: Is it normal that a new feature changed version from stable to SNAPSHOT?

Comment (jnizet) [inline gradle.properties]: I'll change it to 1.6.0 just before releasing.

Comment (clacote) [inline gradle.properties]: Maybe the release task should bump version and add "-SNAPSHOT" as soon as the release is done (as would Maven do).

**Comment by clacote** [comment]
The new API `repeatingValues()` is not documented as exemple in `Insert` javadoc.

File: src/test/java/com/ninja_squad/dbsetup/operation/InsertTest.java [hunk 2]
```diff
@@ -240,6 +241,114 @@ public void endRowFailsWhenItContainsUnknownColumnNameAndColumnNamesSpecifiedByF
...
+        inOrder.verify(bBinder).bind(statement, 2, "b1");
+        inOrder.verify(cBinder).bind(statement, 3, null);
+        inOrder.verify(statement).executeUpdate();
+        inOrder.verify(aBinder).bind(statement, 1, "a1");
+        inOrder.verify(bBinder).bind(statement, 2, "b1");
+        inOrder.verify(cBinder).bind(statement, 3, null);
+        inOrder.verify(statement).executeUpdate();
+        inOrder.verify(statement).close();
+    }
+
```
Comment (clacote) [inline src/test/java/com/ninja_squad/dbsetup/operation/InsertTest.java]: Is it possible to use a `repeatingValues` as first row definition, on a `Builder` without explicitely defined columns? If yes, there seems to lack a test for this.

**Comment by clacote** [comment]
Besides my (uneducated) comments: :+1:!

**Pull request merged**